assets/sass/patterns/atoms/modal-window.scss

```diff
@@ -0,0 +1,138 @@
+.modal-container {
+  background-color: rgba(255,255,255,0.5);
+  height: 100%;
+  left: 0;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  visibility: hidden;
+}
+
+.modal-content {
+  background-color: #fff;
+  border-radius: 3px;
+  box-shadow: 0 0 20px rgba(0, 0, 0, 0.22);
+  display: grid;
+  margin: 0;
+  padding: 36px;
+
+  @media only all and (min-width: #{get-rem-from-px($bkpt-site--small)}em) {
+    margin: 24px;
+  }
+
+  @media only all and (min-width: #{get-rem-from-px($bkpt-site--medium + 48)}em) {
+    margin: 24px auto;
+    max-width: 730px;
+  }
+
+  h6 {
+    @include font-size-and-vertical-height($font-size-h6-in-px);
+    grid-column: 1/3;
+    grid-row: 1;
+    justify-self: start;
+    padding-top: 0;
+  }
+
+  .social-media-sharers {
+    @media only all and (max-width: #{get-rem-from-px($bkpt-site--small - 1)}em) {
+      @include padding(24, "top");
+      display: flex;
+      grid-column: 1/3;
+      grid-row: 4;
+      justify-content: center;
+    }
+  }
+
+  .reference {
+    font-family: $font-secondary;
+    grid-column: 1/3;
+
+    .doi {
+      @include margin(12, "top");
+      @include margin(36, "bottom");
+      display: flex;
+    }
+  }
+
+  .button-collection {
+    display: unset;
+    flex-direction: column;
+    grid-column: 1/3;
+
+    .button-collection__item {
+      float: none;
+    }
+
+    .button {
+      @include margin(12, "bottom");
+      width: 100%;
+    }
+
+    @media only all and (min-width: #{get-rem-from-px($bkpt-site--medium)}em) {
+      grid-column: 1/3;
+
+      .button-collection__item {
+        float: left;
+      }
+
+      .button {
+        @include margin(12, "bottom");
+        line-height: 1.5;
+        padding: 15px 24px 14px;
+        text-decoration: none;
+        width: auto;
+      }
+    }
+  }
+}
+
+.modal-content__body {
+  display: grid;
+  grid-column: 1/3;
+
+  .button {
+    justify-self: center;
+    margin-bottom: 24px;
```

Comment:
There are a few cases where it would be appropriate to use a mixin such as here `@include margin(24, "bottom");`

Reply:
Had not finished this part of the changes